examples/torch/bc_point_deterministic_policy.py

```diff
@@ -0,0 +1,82 @@
+"""Example of using Behavioral Cloning."""
+import numpy as np
+
+from garage import wrap_experiment
+from garage.envs import GarageEnv, PointEnv
+from garage.experiment import LocalRunner
+from garage.torch.algos import BC
+from garage.torch.policies import DeterministicMLPPolicy, Policy
+
+
+class OptimalPolicy(Policy):
+    """Optimal policy for PointEnv.
+
+    Args:
+        env_spec (EnvSpec): The environment spec.
+        goal (np.ndarray): The goal location of the environment.
+
+    """
+
+    # No forward method
+    # pylint: disable=abstract-method
+
+    def __init__(self, env_spec, goal):
+        super().__init__(env_spec, 'OptimalPolicy')
+        self.goal = goal
+
+    def get_action(self, observation):
+        """Get action given observation.
+
+        Args:
+            observation (np.ndarray): Observation from the environment.
+
+        Returns:
+            tuple:
+                * np.ndarray: Action.
+                * dict[str, np.ndarray]: Agent info (empty).
+
+        """
+        return self.goal - observation[:2], {}
+
+    def get_actions(self, observations):
```

Comment:
please document shapes. Variables to describe the shape can be found in a table under garage/_dtypes.py